appveyor.yml

```diff
@@ -5,7 +5,7 @@
 #---------------------------------#
 
 # version format
-version: 3.1.0.{build}
+version: 3.1.{build}

```

Comment:
What is now displayed in the SonarQube Server Marketplace as version? Until now it was e.g. `3.1 (build 345)installed`